modelingMain.py

```diff
@@ -45,4 +45,7 @@
                 cv_result.save_rocs(util.results_path, ds)
                 pickle.dump(cv_result, results_pickle)
                 print("cv pickled")
+            with open("nn_state.pkl", "wb") as f:
```

Comment:
I think this happens six times, once for each label and within that once for the Lap dataset and once for the open dataset, right? We should be saving six different versions of the neural network, e.g. `lap_in_hospital_mortality_predictor.pkl`, `open_length_of_stay_predictor.pkl`, etc.

Reply:
Okay I will make sure that the models are uniquely named.